org.keepassxc.KeePassXC.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "org.keepassxc.KeePassXC",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "keepassxc",
+  "rename-icon": "keepassxc",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Wayland access */
+    "--socket=wayland",
+    /* Access user files */
+    "--filesystem=home",
```

Comment:
Once you enable the flatpak backend linked in my other comment you *may* no longer need this. Just depends how and for what it needs this access.

Reply:
Unfortunately `"--env=QT_QPA_PLATFORM=flatpak",` seems to break the KeePassXC auto-type feature. A lot of users will not accept that, me being one of them.

Do you happen to know where I may report or discuss this issue?

Reply:
On Qts bug tracker probably.

Reply:
Yeah, or the KDE team who work on this and the runtime.

Reply:
the QPA backend isn't required any more - the KDE SDK deals with this automatically. However @aleixpol may still be interested in the issue.

Reply:
Hey, yes at the moment `QT_QPA_PLATFORM` isn't needed and we'll be doing "the right thing™️".
It may be that there's some issue (I'm not familiar with the auto-type feature) but if you can track down where the issue is I'll be happy to look into it.

An bug report can be opened on our bugzilla: https://bugs.kde.org/ to the right component. Please keep me posted.